toolkit/tools/pkg/imagecustomizerlib/configvalidation.go

```diff
@@ -64,6 +64,11 @@ func ValidateConfig(ctx context.Context, baseConfigPath string, config *imagecus
 		return nil, err
 	}
 
+	configChain, err := buildConfigChain(ctx, rc)
```

Comment:
Store the `ConfigChain` in `ResolvedConfig`. It'll be needed by the features that walk the chain (e.g. packages).